ansible/vars/topo_t0.yml

```diff
@@ -50,6 +50,26 @@ topology:
       vlans:
         - 31
       vm_offset: 3
+  DUT:
+    vlan_cfgs:
+      default_vlan_cfg: one_vlan_a
+      one_vlan_a:
+        Vlan1000:
```

Comment:
what is _a?

Reply:
In case of multiple one vlan configuration like some ports are not part of any vlan configuration, would take 'b' as suffix. That is the idea behind it. suggestion?

Reply:
That was a question :)